src/components/SideMenu/index.tsx

```diff
@@ -0,0 +1,77 @@
+import Link from 'next/link' // Import Next.js Link
+import { CaretUpIcon } from '@components/Icon'
+import { useState } from 'react'
+import ROUTE from '@constants/route'
+
+const menus = [
+  {
+    header: 'Features',
+    items: [
+      { name: 'Instagram Trend Analysis', path: ROUTE.DASHBOARD },
+      { name: 'Manage Instabot Accounts', path: ROUTE.ACCOUNTS },
+      { name: 'Hashtags-to-Image', path: ROUTE.HASHTAG_TO_IMAGE },
+      { name: 'Image-to-Hashtags', path: ROUTE.IMAGE_TO_HASHTAG },
+    ],
+  },
+  {
+    header: 'User Settings',
+    items: [
+      { name: 'Profile Settings', path: ROUTE.ACCOUNTS },
+      { name: 'Privacy Settings', path: ROUTE.ACCOUNTS },
+    ],
+  },
+  {
+    header: 'Support',
+    items: [
+      { name: 'Contact', path: ROUTE.ACCOUNTS },
+      { name: 'FAQ', path: ROUTE.ACCOUNTS },
+    ],
+  },
+]
+
+const SideMenu = () => {
+  const [active, setActive] = useState<string>('')
+
+  const toggleMenu = (header: string) => {
+    setActive(active === header ? '' : header) // Toggle active state
+  }
+
+  return (
+    <div className="flex h-full max-h-screen w-full flex-col justify-between rounded-lg border border-neutral-300 p-4 text-base">
+      <div>
+        {menus.map((menu) => (
+          <div key={menu.header} className="dropdown">
+            <button
+              className={`dropdown-toggle flex w-full flex-row justify-between px-4 py-2 font-bold hover:text-primary-500 ${
+                active === menu.header ? 'active text-primary-500' : ''
+              }`}
+              onClick={() => toggleMenu(menu.header)} // Handle click to toggle dropdown
```

Comment:
https://stackoverflow.com/questions/64425063/how-to-avoid-anonymous-function-in-react